charger/ocpp.go

```diff
@@ -217,6 +217,8 @@ func NewOCPP(id string, connector int, idtag string, meterValues string, meterIn
 		c.meterValuesSample = meterValues
 	}
 
+	c.log.DEBUG.Printf("currently sampled meter values: %s \r\n", c.meterValuesSample)
```

Comment:
The entire configuration is already logged above- why is this needed?

Reply:
Shows final value and is level higher (debug instead of trace) so it is easier to spot in the noise when someone doesn't know what they should put into metervalues or their charger is misconfigured (my case), but can be omitted if you don't find it useful.